frontend/src/App.tsx

```diff
@@ -1,28 +1,23 @@
-import {useState} from 'react';
-import logo from './assets/images/logo-universal.png';
-import './App.css';
-import {Greet} from "wailsjs/go/main/App";
+import { useState } from 'react';
 
-function App() {
-    const [resultText, setResultText] = useState("Please enter your name below 👇");
-    const [name, setName] = useState('');
-    const updateName = (e: any) => setName(e.target.value);
-    const updateResultText = (result: string) => setResultText(result);
+import { Button } from './components/atomic';
+import { Line } from './components/molecule';
 
-    function greet() {
-        Greet(name).then(updateResultText);
-    }
+function App() {
+  const d: Array<number> = [1, 25, 3];
```

Comment:
That's a `ReadonlyArray`. plus, you don't have to store your array in an extra variable.